splice_machine/src/test/java/com/splicemachine/derby/impl/load/HBaseBulkLoadIT.java

```diff
@@ -101,22 +101,23 @@ public static void cleanup() throws Exception {
     private static void testBulkDelete() throws Exception {
         if (notSupported)
             return;
-        bulkDelete(LINEITEM);
-        bulkDelete(ORDERS);
-        bulkDelete(CUSTOMERS);
-        bulkDelete(PART);
-        bulkDelete(PARTSUPP);
-        bulkDelete(NATION);
-        bulkDelete(REGION);
-        bulkDelete(SUPPLIER);
+        bulkDelete(LINEITEM, "L_PART_IDX");
+        bulkDelete(ORDERS, "O_CUST_IDX");
+        bulkDelete(CUSTOMERS, null);
+        bulkDelete(PART,null);
+        bulkDelete(PARTSUPP,null);
+        bulkDelete(NATION,null);
+        bulkDelete(REGION,null);
+        bulkDelete(SUPPLIER,null);
         countUsingIndex(LINEITEM, "L_PART_IDX");
         countUsingIndex(LINEITEM, "L_SHIPDATE_IDX");
         countUsingIndex(ORDERS, "O_CUST_IDX");
         countUsingIndex(ORDERS, "O_DATE_PRI_KEY_IDX");
     }
 
-    private static void bulkDelete(String tableName) throws Exception {
-        String sql = String.format("delete from %s --splice-properties bulkDeleteDirectory='%s'", tableName, getResource("data"));
+    private static void bulkDelete(String tableName, String indexName) throws Exception {
+        String sql = String.format("delete from %s --splice-properties bulkDeleteDirectory='%s',index=%s", tableName, getResource("data"),
+                indexName!=null?indexName:"null");
```

Comment:
Could you also add IT for regular delete with index hint?

Reply:
Regular delete was not impacted.

Reply:
It only happens when using results from index lookup to create index using IndexTransformer. Bulk delete is the only case